### PR TITLE
Task 1 and 2

### PR DESCRIPTION
**Pull request by wwwIgorNet**

### CODE REVIEW

File: Cart Service/WebApi/WebApi.csproj [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <Nullable>enable</Nullable>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <AssemblyName>$(MSBuildProjectName)</AssemblyName>
+    <RootNamespace>OnlineShopping.CartService.$(MSBuildProjectName.Replace(" ", "_"))</RootNamespace>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AutoMapper" Version="13.0.1" />
```
Comment (rkosmirak) [inline Cart Service/WebApi/WebApi.csproj:12]: [Optional] If you would like to try something interesting consider using [Central package management](https://learn.microsoft.com/en-us/nuget/consume-packages/central-package-management)

File: Catalog Service/Catalog Service.sln
Comment (rkosmirak) [inline Catalog Service/Catalog Service.sln:1]: [Optional] If you would like to try something interesting consider using [slnx](https://medium.com/c-sharp-programming/a-closer-look-at-visual-studios-new-slnx-file-format-521e15a20e96)

File: Catalog Service/InfrastructureTest/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
+namespace InfrastructureTest
+{
+    public class Tests
```
Comment (rkosmirak) [inline Catalog Service/InfrastructureTest/UnitTest1.cs]: probably you don't need it

File: Catalog Service/Infrastructure/Data/Configurations/CategoryConfiguration.cs [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+        builder.HasKey(c => c.Id);
+
+        builder.HasMany(c => c.Products)
+            .WithOne(p => p.Category)
+            .HasForeignKey(p => p.CategoryId)
+            //.HasPrincipalKey(c => c.Id)
+            .IsRequired()
+            .OnDelete(DeleteBehavior.Cascade);
+
+        //builder.HasMany(c => c.ChildCategories)
```
Comment (rkosmirak) [inline Catalog Service/Infrastructure/Data/Configurations/CategoryConfiguration.cs]: do you need it?

File: Catalog Service/Application/Application.csproj [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    <PackageReference Include="Ardalis.GuardClauses" Version="5.0.0" />
+    <PackageReference Include="AutoMapper" Version="13.0.1" />
+    <PackageReference Include="MediatR" Version="12.4.1" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\Domain\Domain.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
```
Comment (rkosmirak) [inline Catalog Service/Application/Application.csproj]: consider removing this just to keep csproj files clean (vs adding this here automatically when you are creating folder, but unfortunately, not able to remove this automatically)

**Review by rkosmirak — APPROVED** [review]

File: Cart Service/WebApi/WebApi.csproj (hunk 1 above)
Comment (wwwIgorNet) [reply Cart Service/WebApi/WebApi.csproj:12]: Thanks, I will try it in next module

**Pull request merged**